### PR TITLE
Try updating omniauth-gumroad's version number and also update omniauth

### PR DESCRIPTION
**Pull request by BenSandeen**
Just flailing about, trying to get things to upgrade

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -13,26 +13,31 @@ GEM
       safe_yaml (~> 0.9.0)
     diff-lcs (1.2.5)
     docile (1.1.2)
-    faraday (0.17.1)
-      multipart-post (>= 1.2, < 3)
-    hashie (3.6.0)
-    jwt (2.2.1)
+    faraday (2.7.4)
```
Comment (BenSandeen) [inline Gemfile.lock:16]: 😭

File: .gitignore [hunk 2]
```diff
@@ -1,18 +1,56 @@
...
+/.bundle/
+/vendor/bundle
+/lib/bundler/man/
+
+# for a library or gem, you might want to ignore these files since the code is
+# intended to run in multiple environments; otherwise, check them in:
+# Gemfile.lock
+# .ruby-version
+# .ruby-gemset
+
```
Comment (BenSandeen) [inline .gitignore]: I copied this file from https://github.com/github/gitignore/blob/main/Ruby.gitignore

Should we not be tracking the gemfile in this repo, then?  I'm not familiar with how all this plays together

Comment (atongen) [reply .gitignore]: There is a lot of stuff in that gitignore that isn't relevant to us. I'd be hesitant to pull it in in its entirety.

We can probably ignore `Gemfile.lock`, but then we should probably put version constrains in `omniauth-gumroad.gemspec` to match the upgrades we're trying to achieve in the monolith.

Comment (atongen) [reply .gitignore]: ie. around here: https://github.com/DripEmail/omniauth-gumroad/blob/master/omniauth-gumroad.gemspec#L19-L20

**Comment by atongen** [comment]
I also just noticed https://github.com/DripEmail/omniauth-gumroad/pull/1 (sorry I'm late!). Typically you would not edit the lockfile directly. You'd specify your version constrains in `Gemfile` (or `.gemspec` for a gem), and then let bundler's dependency solver write the lockfile for you.

**Comment by BenSandeen** [comment]
> I also just noticed #1 (sorry I'm late!). Typically you would not edit the lockfile directly. You'd specify your version constrains in `Gemfile` (or `.gemspec` for a gem), and then let bundler's dependency solver write the lockfile for you.

Thanks for the tip, I wasn't sure what the `.gemspec` was for, I was thinking it was just an alternative way of managing dependencies.  I can't recall if I did that for a good reason, or if I just wanted to do the heavy-handed thing that I knew would get the job done :/

File: .gitignore (hunk 2 above)
Comment (BenSandeen) [reply .gitignore]: Yeah, thanks for pushing back, I just decided to add a line to ignore the `vendor/` directory.  And I'm reading up on the gemspec stuff to understand how we can use that 👍

**Comment by BenSandeen** [comment]
Closing in favor of https://github.com/DripEmail/omniauth-gumroad/pull/3

**Pull request closed without merging**